app/assets/javascripts/attractions.js

```diff
@@ -0,0 +1,39 @@
+$(document).ready(function(){
+
+  $('#add-attraction').click(function(event){
+    event.preventDefault();
+
+    var url = $(this).attr('href');
+
+    $.ajax({
+      url: url,
+      type: "get",
+      beforeSend: function(xhr) { xhr.setRequestHeader('X-CSRF-Token', $('meta[name=csrf-token]').attr('content'))},
```

Comment:
Assuming you have multiple ajax requests, I'd recommend you use this (once) instead:

```javascript
$.ajaxSetup({
  headers: {
    'X-CSRF-Token': $('meta[name=csrf-token]').attr('content')
  }
});
```